### PR TITLE
[Fundamental] Content re-org + migrated a KB

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/_redirects [hunk 1]
```diff
@@ -244,14 +244,20 @@
 /firewall/cf-rulesets/view-rulesets /ruleset-engine/basic-operations/view-rulesets 301
 
 # fundamentals
-/fundamentals/notifications/configure-webhooks /fundamentals/notifications/create-notifications/configure-webhooks 301
```
Comment (kodster28) [inline content/_redirects:247]: added ending slashes to these and moved them into more alphabetical order.

File: content/_redirects [hunk 2]
```diff
@@ -244,14 +244,20 @@
...
-/fundamentals/notifications/create-notification /fundamentals/notifications/create-notifications/create-notification 301
-/fundamentals/notifications/create-pagerduty /fundamentals/notifications/create-notifications/create-pagerduty 301
-/fundamentals/notifications/edit-notification /fundamentals/notifications/edit-notifications/edit-notification 301
-/fundamentals/notifications/edit-pagerduty /fundamentals/notifications/edit-notifications/edit-pagerduty 301
-/fundamentals/notifications/edit-webhook /fundamentals/notifications/edit-notifications/edit-webhook 301
 /fundamentals/data-products/accessing-cf-data/ /fundamentals/data-products/types-of-analytics/ 301
-/fundamentals/data-products/accessing-cf-data /fundamentals/data-products/types-of-analytics/ 301
+/fundamentals/get-started/cdn/ /fundamentals/get-started/concepts/cdn/ 301
```
Comment (kodster28) [inline content/_redirects:248]: These get started redirects are new

File: content/ssl/ssl-for-saas/reference/versioning.md [hunk 3]
```diff
@@ -34,7 +34,9 @@ When creating a Custom Hostname with the HTTP through the [API](https://api.clou
 
 Cloudflare sends GET requests to the http_url using `User-Agent: Cloudflare Custom Hostname Verification`.
 
-When the HTTP token is verified, the Custom Hostname will be marked as **Active** in the Cloudflare SSL/TLS app under the Custom Hostnames tab.
+If you validating a hostname that is not proxying traffic through Cloudflare, the Custom Hostname will be marked as **Active** in the Cloudflare SSL/TLS app when the HTTP token is verified (under the **Custom Hostnames** tab).
```
Comment (marciocloudflare) [inline content/ssl/ssl-for-saas/reference/versioning.md]:
```suggestion
If you validated a hostname that is not proxying traffic through Cloudflare, the Custom Hostname will be marked as **Active** in the Cloudflare SSL/TLS app when the HTTP token is verified (under the **Custom Hostnames** tab).
```

Comment (marciocloudflare) [reply content/ssl/ssl-for-saas/reference/versioning.md]: Not sure if this was the verb tense you were going for. Please check.

**Review by marciocloudflare — APPROVED** [review]

**Comment by pedrosousa** [comment]
These changes are causing new CF Pages deployment warnings about redirects. There's this one:
```
Found invalid redirect lines:
    Maximum number of dynamic rules supported is 100. Skipping remaining 694 lines of file.`
```
And a few similar to this one:
```
Info: the redirect rule /fundamentals/notifications/configure-webhooks/ → 301 /fundamentals/notifications/create-notifications/configure-webhooks/ could be made more performant by bringing it above any lines with splats or placeholders.
```

**Pull request merged**